index.html

```diff
@@ -1,29 +1,152 @@
 <!DOCTYPE html>
-<html lang="en-ca">
-  <head>
-    <!-- Set the page title -->
-    <title>Assignment 02</title>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, intial-scale=1.0">
+        <Title>House of Shelby</Title>
+        <link href="style.css"rel="stylesheet" type="text/css">
+        <link href="reset.css"rel="stylesheet" type="text/css">   
+    </head>
+    <body>
+        <style>
+            body{
+                background-color:#100D0B;
+            }
+            
+        </style>
+        
+        <!--Navigation options for the website-->
+        <div class="Heading">
+            <p><a href="#">menu</a></p>
+            <p><a href="#">process</a></p>
+            <p><a href="#">barrels</a></p>
+            <!--Logo that will be used for the website-->
+            <div class="logo">
+                <img src="./images/shelby.png" alt="Logo Picture is here.">
+            </div>
+            
+            <p><a href="#">visit us</a></p>
+            <p><a href="#">our story</a></p>
+            <p><a href="#">family recipe</a></p>
+        </div>
```

Comment:
This should be marked up as a list of links because, semantically, it's a list of links, and should be inside a `nav` element.

```html
<nav>
  <ul class="nav">
    <li><a href="#">menu</a></li>
    ...
  </ul>
</nav>
```